### PR TITLE
💎 Julius on the team!

### PR DESCRIPTION
**Pull request by tallys**
💎 
Copied from Slack - so should be: 

name: Julius Yanik
handle: @juliusschaeper
title: Systems Designer
What drew you into design systems?
Being able to focus on the key elements of design, form, color and typography.
Who have you learned from or been inspired by?
Having an education in industrial design, the Bauhaus design philosophy and Dieter Rams inspired me a lot. Everything I know about digital product design I learned at Wunderlist, which design team was very inspiring.
Favorite tools?
Remarkable tablet, Figma, Origami, Glyphs, GitHubs internal color tool.

### CODE REVIEW

File: src/data/team.js [hunk 1]
```diff
@@ -190,6 +190,18 @@ const teamContent = [
     gif: 'https://user-images.githubusercontent.com/54012/50536666-fced1680-0b56-11e9-89e6-995cdb01a599.gif',
     alumni: false
   },
+  {
+    name: 'Julius Yanik',
+    title: 'Systems Designer',
+    handle: 'juliusschaeper',
+    cssFeature: `grid-template-areas: "header header header" "footer footer footer"`,
+    questionOne: `Being able to focus on the key elements of design, form, color and typography.`,
+    questionTwo: `Having an education in industrial design, the Bauhaus design philosophy and Dieter Rams inspired me a lot. Everything I know about digital product design I learned at Wunderlist, which design team was very inspiring.`,
```
Comment (yaili) [inline src/data/team.js:199]:
```suggestion
    questionTwo: `Having an education in industrial design, the Bauhaus design philosophy and Dieter Rams inspired me a lot. Everything I know about digital product design I learned at Wunderlist, whose design team was very inspiring.`,
```

**Review by yaili — CHANGES_REQUESTED** [review]
Just a small word change, but I have a question: we're supposed to add ourselves to the top of the list (so most recent team addition first), is that the order Julius's name will show up?

**Comment by tallys** [comment]
Oh! I didn't know there was a rule to the order, yeah! I will rearrange myself as well :)

**Review by colebemis — APPROVED** [review]

**Review by yaili — APPROVED** [review]
🚢

**Pull request merged**